doc/diffs.md

```diff
@@ -0,0 +1,163 @@
+                                crossgen or clrjit from this directory,
+                                depending on whether --crossgen is
+                                specified.
+        -d, --diff <arg>        The diff compiler directory or tag. Will use
+                                crossgen or clrjit from this directory,
+                                depending on whether --crossgen is
+                                specified.
+        --crossgen <arg>        The crossgen compiler exe. When this is
+                                specified, will use clrjit from the --base
+                                and --diff directories with this crossgen
+        -o, --output <arg>      The output path.
+        -a, --analyze           Analyze resulting base, diff dasm
+                                directories.
+        -s, --sequential        Run sequentially; don't do parallel
+                                compiles.
+        -t, --tag <arg>         Name of root in output directory.  Allows
+                                for many sets of output.
+        -c, --corlibonly        Disasm *CorLib only
+        -f, --frameworksonly    Disasm frameworks only
+        --benchmarksonly        Disasm core benchmarks only
+        -v, --verbose           Enable verbose output
+        --core_root <arg>       Path to test CORE_ROOT.
+        --test_root <arg>       Path to test tree.
+```
+
+The "list" command has this help:
+```
+    $ jit-diff diff --help
+    usage: jit-diff list [-v]
+
+        -v, --verbose    Enable verbose output
+```
+
+The "install" command has this help:
+```
+    $ jit-diff install --help
+    usage: jit-diff install [-j <arg>] [-n <arg>] [-l] [-b <arg>] [-v]
+
+        -j, --job <arg>          Name of the job.
+        -n, --number <arg>       Job number.
+        -l, --last_successful    Last successful build.
+        -b, --branch <arg>       Name of branch.
+        -v, --verbose            Enable verbose output
+```
+
+Sample usage, to create diffs:
+```
+    c:\gh\coreclr> jit-diff diff -o c:\diffs -t 5 -a -f --core_root c:\gh\coreclr\bin\tests\Windows_NT.x86.checked\Tests\Core_Root -b e:\gh\coreclr2\bin\Product\Windows_NT.x86.checked -d c:\gh\coreclr\bin\Product\Windows_NT.x86.checked
+```
+
+Explanation:
+1. `-o c:\diffs` -- specify the root directory where diffs will be placed.
+2. `-t 5` -- give the diffs a "tag". Thus, the actual root directory will be `c:\diffs\5`.
+3. `-a` -- run jit-analyze on the resultant diffs.
+4. `-f` -- generate diffs over the framework assemblies (a well-known list built in to jit-diff).
+5. `--core_root` -- specify the `CORE_ROOT` directory created by running `tests\runtest.cmd`
+   or `tests\runtest.cmd GenerateLayoutOnly` in the dotnet/coreclr repo.
+6. `-b` -- specify the directory in which a baseline crossgen.exe can be found.
+7. `-d` -- specify the directory in which a diff (experimental) crossgen.exe can be found.
```

Comment:
Yes I believe so.  Sven added it and it makes clear what "look beside me for System.Private.Corelib" crossgen we mean to use.  Passing SPC on the commandline does not work crossgen still does the look aside.